blog_post.md

```diff
@@ -1,52 +1,229 @@
-to RabbitMQ.
-
-TODO: detail [jobs.yaml](/jobs.yaml) and [service-binding.yaml](/service-binding.yaml)
-
-## An even easier way
-
-Set the label `service.binding/provisioned-service=true` on the custom resource
-(instead of the annotations we would usually set) and everything should work.
-
-TODO: (insert details of it working here)
-
-## The easiest way
-
-The RabbitMQ operator is a part of the service binding registry, so these
-annotations/labels that we set earlier don't even need to be set for this to
-work.
+Let's say I have two kubernetes services, `producer` and `consumer`, that talk
+to a RabbitMQ instance using AMQP.  `producer` periodically produces data that
+`consumer` reads and acts on.  For the sake of this demonstration, that action
+is printing whatever it receives to `stdout`.
+
+We'll return to the concept of binding once we have everything setup.  For now,
+let's get our RabbitMQ cluster setup on a local kubernetes cluster (I prototype
+with `minikube`, but the instructions would be the same if you were to run this
+on an OpenShift cluster).
+
+First, we can setup the RabbitMQ operator:
+```bash
+kubectl apply -f https://github.com/rabbitmq/cluster-operator/releases/latest/download/cluster-operator.yml
+```
+
+Next, we'll need to install Operator Lifecycle Manager (OLM), a prerequisite
+for the Service Binding Operator:
+```bash
+curl -sL https://github.com/operator-framework/operator-lifecycle-manager/releases/download/v0.19.1/install.sh | bash -s v0.19.1
+```
+
+NOTE: yes, doing `curl ... | bash` isn't the best in terms of security.  If this
+is a concern for you, you can instead save the installation script to a location
+in your filesystem and execute the script from there after inspecting its
+contents.
+
+We'll also need the Service Binding Operator:
+```bash
+kubectl apply -f https://operatorhub.io/install/service-binding-operator.yaml
+```
+
+Next, we want to have our `producer` and `consumer` running on our kubernetes
+cluster.  For convenience, I've authored two containers that provide this
+functionality; their sources can be found
+[here](https://github.com/sadlerap/sbo-rabbitmq-sample).
+
+SBO can operate against deployments, and deployments make the most sense for
+running our applications.  You can deploy them with the following:
+```bash
+kubectl create deployment producer --image=quay.io/ansadler/rabbitmq-producer:latest
+kubectl create deployment consumer --image=quay.io/ansadler/rabbitmq-consumer:latest
+```
+
+We'll also want a rabbitmq cluster to run them against:
+```yaml
+apiVersion: rabbitmq.com/v1beta1
+kind: RabbitmqCluster
+metadata:
+  name: rabbitmq
+spec:
+  service:
+    type: ClusterIP
+```
+
+If you wish, you can deploy all of these (that is, `producer`, `consumer`, and
+a rabbitmq cluster) with the following:
+```bash
+kubectl apply -f https://raw.githubusercontent.com/sadlerap/sbo-rabbitmq-sample/master/jobs.yaml
+```
```

Comment:
why offering another method to deploy them when you above do that with two kubectl commands? what is the advantage?